03-vectors/vectors-practice-2.Rmd

```diff
@@ -72,6 +92,18 @@ draw(p2x, p2y)
 
 here, we almost have four duplicates of Hobbes in a square configuration, but the upper right drawing is missing. Please duplicate one of the drawings of Hobbes, and place it in the upper right corner (by modifying the `p2x` and `p2y` vectors, that is).
 
+```{r}
+new.p2x = p2x
+new.p2y = p2y
+# Select a subset of points that includes the upper-left Hobbes and move them to the upper-right
+copy.x = new.p2x[new.p2x < 50 & new.p2y > 50] + 50
+copy.y = new.p2y[new.p2x < 50 & new.p2y > 50]
+# Add the points to the vectors
+new.p2x = c(new.p2x, copy.x)
+new.p2y = c(new.p2y, copy.y)
+# Draw
+draw(new.p2x, new.p2y)
+```
 1.   
```

Comment:
Ah! I was trying to do this with in line substitution and it was getting really messy. I like that you just added new data points.